styles.css

```diff
@@ -0,0 +1,174 @@
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+/*    colors*/
+    :root{
+       --paars:#5722CD;
+       --blauw:#4845E4;
+       --roze:#ED4385;
+       --zwart:#0C0C0C;
+       --lichtpaars:#F2F0FF;
+       --lichtgrijs:#F3F6F9;
+       --grijs:#7A7A7A;
+       /*--backgroundgrad:rgb(75,73,231);*/
+       /* background: linear-gradient(135deg, rgba(75,73,231,1) 60%, rgba(96,44,207,1) 100%);*/
+    }
+/* ------------------------------- 2. Typography ----------------------------------- */
+    @font-face {
+        src: url("assets/fonts/Merriweather-Black.ttf");
+        font-family: Merri;
+        font-weight: normal;
+    }
+    @font-face {
+        src: url("assets/fonts/Merriweather-Bold.ttf");
+        font-family: Merri;
+        font-weight: bold;
+    }
+    @font-face {
+        src: url("assets/fonts/Merriweather-light.ttf");
+        font-family: Merri;
+        font-weight: 200;
+    }
+    @font-face {
+        src: url("assets/fonts/Roboto-Black.ttf");
+        font-family: robo;
+        font-weight: normal;
+    }
+    @font-face {
+        src: url("assets/fonts/Roboto-Bold.ttf");
+        font-family: robo;
+        font-weight: bold;
+    }
+    @font-face {
+        src: url("assets/fonts/Roboto-Thin.ttf");
+        font-family: robo;
+        font-weight: 200;
+    }
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+/*    .test{*/
+/*        background:var(--roze);*/
+/*    }*/
+    button{
+        background-color: var(--roze);
+        font-family: robo, serif;
+        font-weight: 400;
+        color:var(--lichtpaars);
+        border-radius: 40px;
+        border-style: none;
+        padding: 15px 20px;
+        margin: 0px 225px;
```

Comment:
Deze margin is niet nodig als je alles (samen met de button) in de header laat centreren (zie andere comment)